examples/gaia-fast-list/app.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var debug = 0 ? (...args) => console.log('[GaiaFastListDemo]', ...args) : ()=>{};
+
+var list = document.querySelector('gaia-fast-list');
+
+list.configure({
+  getSectionName: item => {
+    return item.date;
+  },
+
+  itemKeys: {
+    title: 'title',
+    body: 'metadata.body',
+    image: 'image',
+    link: 'link'
+  },
+});
+
+var chunkSize = 100;
+var count = 0;
+
+function loadNext() {
+  getDataAsync(count, chunkSize).then(data => {
+    debug('got data', data);
+    list.model = list.model.concat(data);
+    count += chunkSize;
+  });
+}
+
+function getDataAsync(from, limit) {
+  debug('get data async', from, limit);
+  var total = 350;
```

Comment:
do we want to move `var total = 350;` at the level of `chunkSize` and `count` and add a 
```
    if (count < total) {
      loadNext();
    }
```
at the end of `loadNext` ?